### ISSUE TITLE
support for POS printer / Therminal printer for remote MCDU

### ISSUE THREAD

**Issue by andrejetcat** [flybywiresim/a32nx#6443]
### Aircraft Version

Development

### Description

I have a Terminal/Reciep Printer (Netum NT-1809DD) connected with my PC.

When I want to pring a normal Notepad dokument, it works well. Also normal Word documents are well printed. 
But if I select the POS Printer with the MCDU interface and hit the "print" button, only a pice of white paper comes out of the POS. 
The Print function only works with "normal" DinA4 printers i think!?
Maybe a format-thing?

It would be awesome, if we could print METARS/TAFs/ATIS etc via a small terminal printer.

**Comment by tyler58546** [comment]
Can your printer print PDFs?

**Comment by andrejetcat** [comment]
no, only word, and wordpad dokuments.
Does the MCDU server create a PDF document for printing?

**Comment by tracernz** [comment]
It works with my HP A799, although better with the font size tweaked a little larger.

There is also a raw mode for these printers which works best but I didn’t find a good node library for this… they’re all broken or unsuitable in different ways.

**Comment by andrejetcat** [comment]
I think the simplest way is to sent out the printer command as a simple .txt/.odt document for those kinds of printers. Is that possible?

**Comment by tyler58546** [comment]
> Does the MCDU server create a PDF document for printing?

Yes, in fact you can find the last generated one at `%TEMP%\a32nxPrint.pdf`.

**Comment by andrejetcat** [comment]
I found the document but my 58mm terminal printer do not print .pdf´s. Maybe there could be an option to create normal word dokuments for the printouts.
@tracernz did you set your printer for the pdf´s?

**Comment by tracernz** [comment]
> I found the document but my 58mm terminal printer do not print .pdf´s. Maybe there could be an option to create normal word dokuments for the printouts. @tracernz did you set your printer for the pdf´s?

No, to my surprise it just worked.

**Comment by ssewell** [comment]
Something seems off. The Netum site shows examples of PDF printing and the print drivers should rasterize the file contents before sending it to the printer. Do you have the Netum drivers currently installed? What happens if you try to open a PDF and print it directly (outside of the MCDU server)?

**Comment by andrejetcat** [comment]
@ssewell yes, i reinstalled the driver several times. Same happends, if I manually print the .pdf document from the %TEMP%\a32nxPrint.pdf file. Then, the problem is on my end and maybe I will try another printer. Thank you for your help :)

**Comment by andrejetcat** [comment]
> It works with my HP A799, although better with the font size tweaked a little larger.
> 
> There is also a raw mode for these printers which works best but I didn’t find a good node library for this… they’re all broken or unsuitable in different ways.

@tracernz  how do you make the font size bigger? i try to set a new in the SumatraPDF-settings.txt, but it seems to be the wrong. 
Maybw my printer works, when i select "print as photo" instead of "print as text. 
I will try it with a bigger font size
